plugins/auth0/pkg/fetch/fetch.go

```diff
@@ -170,3 +188,87 @@ func (f *Fetcher) getRoles(ctx context.Context, uID string) ([]map[string]interf
 	}
 	return results, nil
 }
+
+type User struct {
+	management.User
+}
+
+type UserList struct {
+	management.List
+	Users []*User `json:"users"`
+}
+
+func (ul UserList) UserList() []*management.User {
+	mul := []*management.User{}
+	mul = append(mul, lo.Map(ul.Users, func(v *User, i int) *management.User { return &v.User })...)
```

Comment:
You may have written it this way to make the type clearer but you could write this entire function as:
```go
    return  lo.Map(ul.Users, func(v *User, i int) *management.User { return &v.User })
```

Reply:
Nice, updated, thank you!